src/PowerManifold.jl

```diff
@@ -209,8 +209,12 @@ function allocate_result(M::PowerManifoldNested, f, x...)
         ]
     end
 end
-function allocate_result(::PowerManifoldNestedReplacing, f, x...)
-    return copy(x[1])
```

Comment:
Same question as above – just curious :)

Reply:
What was failing before is `allocate_result(PowerManifold(...), rand)`, because there was no `x[1]`.

Reply:
Ah! sure – that makes sense.